setEvent.php

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="conference.css" type="text/css" rel="stylesheet" />
+    </head>
+    <body>
+        <?php
+            //This page is used to pass information to the database recieving form data from getSelect.php 
+            //notifying the user on its success or failure.
+            include './pdo.php';
+            //Variables passed by the user in a post method from the form created in getSelects.php
+            $day = $_POST["sessionday"];
+            $time = $_POST["sessiontime"];
+            $room = $_POST["sessionroom"];
+            $session = $_POST["session"]; //information used to identify the unique event
+            $output = array();
+            parse_str($session, $output);
+            $origTime = $output["startTime"];
+            $origRoom = $output["room"];
+            $origName = $output["name"];
+            //Queries the database looking for duplicates of the event
+            $query = "SELECT * FROM SessionEvent WHERE sessionName='$origName' AND room='$room' AND startTime=FROM_UNIXTIME($time);";
+            $stmt = $pdo->prepare($query);
+            $stmt->execute();
+
+            //If not in the database
+            if ($stmt->rowCount() <= 0){
```

Comment:
There's still a couple of issues here.
When I queried for something to be changed to 4PM, I got back 11AM instead